config/env.ini

```
@@ -81,25 +81,27 @@ SPC_MICRO_PATCHES=static_extensions_win32,cli_checks,disable_huge_page,vcruntime
 ; buildconf command
 SPC_CMD_PREFIX_PHP_BUILDCONF="./buildconf --force"
 ; configure command
-SPC_CMD_PREFIX_PHP_CONFIGURE="./configure --prefix= --with-valgrind=no --enable-shared=no --enable-static=yes --disable-all --disable-cgi --disable-phpdbg --with-pic"
+SPC_CMD_PREFIX_PHP_CONFIGURE="./configure --prefix= --with-valgrind=no --disable-shared --enable-static --disable-all --disable-cgi --disable-phpdbg --with-pic"
 ; make command
 SPC_CMD_PREFIX_PHP_MAKE="make -j${CPU_COUNT}"
 ; embed type for php, static (libphp.a) or shared (libphp.so)
 SPC_CMD_VAR_PHP_EMBED_TYPE="static"
 
 ; *** default build vars for building php ***
 ; CFLAGS for configuring php
-SPC_CMD_VAR_PHP_CONFIGURE_CFLAGS="${SPC_DEFAULT_C_FLAGS} -fPIE"
+SPC_CMD_VAR_PHP_CONFIGURE_CFLAGS="${SPC_DEFAULT_C_FLAGS} -fpie"
```

Seems we don't need fpie here.
```suggestion
SPC_CMD_VAR_PHP_CONFIGURE_CFLAGS="${SPC_DEFAULT_C_FLAGS}"
```

---

We should definitely keep it there. It should be present in both configure (to create the makefile) and make.

---

SPC_DEFAULT_C_FLAGS includes -fpic, not -fpie

---

-fpic should be used for the creation of code/static libraries. -fpic should only be used for the creation of executables.